docker/entrypoint.sh

```diff
@@ -27,11 +27,18 @@ function init_if_possible () {
 
 function generate_password () {
     set +o pipefail
-    < /dev/urandom tr -dc A-Za-z0-9 | head -c40
+    < /dev/urandom tr -dc A-Za-z0-9 | head -c256
     echo
     set -o pipefail
 }
 
+function set_password_files () {
+    local file_path="$1"
```

Comment:
Can you please make your bash variables in the following format:

```bash
FOO=bar
echo "${FOO}"
```

Declare them in all caps and then when referencing them enclose them in quotations and squiggly brackets.

Reply:
force pushed fixing everything